mcs/class/Facades/netstandard/Makefile

```diff
@@ -28,32 +28,29 @@ endif
 ifeq ($(PROFILE),monodroid)
 # XA implements System.Drawing API inside Mono.Android
 API_BIN_REFS := Mono.Android
-else
+endif
+
+ifeq ($(PROFILE),monotouch)
+API_BIN_REFS := Xamarin.iOS OpenTK-1.0
+endif
+
+ifeq ($(PROFILE),monotouch_tv)
+API_BIN_REFS := Xamarin.TVOS OpenTK-1.0
+endif
+
+ifeq ($(PROFILE),monotouch_watch)
+API_BIN_REFS := Xamarin.WatchOS
+endif
+
+ifeq ($(PROFILE),xammac)
+API_BIN_REFS := Xamarin.Mac OpenTK-1.0
```

Comment:
this is unfortunately still wrong for XM, it uses OpenTK.dll (NOT OpenTK-1.0.dll !) for Color and KnownColor :)

Reply:
oh, missed that one again!